src/components/Typography/src/typography.module.css

```diff
@@ -0,0 +1,53 @@
+.denhaag-heading-1 {
+  font-size: var(--denhaag-heading-1-font-size);
+  font-family: var(--denhaag-heading-1-font-family);
+  font-weight: var(--denhaag-heading-1-font-weight);
+  font-style: var(--denhaag-heading-1-font-style);
+  line-height: var(--denhaag-heading-1-line-height);
+  letter-spacing: var(--denhaag-heading-1-letter-spacing);
+}
+
+.denhaag-heading-2 {
+  font-size: var(--denhaag-heading-2-font-size);
+  font-family: var(--denhaag-heading-2-font-family);
+  font-weight: var(--denhaag-heading-2-font-weight);
+  font-style: var(--denhaag-heading-2-font-style);
+  line-height: var(--denhaag-heading-2-line-height);
+  letter-spacing: var(--denhaag-heading-2-letter-spacing);
+}
+
+.denhaag-heading-3 {
+  font-size: var(--denhaag-heading-3-font-size);
+  font-family: var(--denhaag-heading-3-font-family);
+  font-weight: var(--denhaag-heading-3-font-weight);
+  font-style: var(--denhaag-heading-3-font-style);
+  line-height: var(--denhaag-heading-3-line-height);
+  letter-spacing: var(--denhaag-heading-3-letter-spacing);
+}
+
+.denhaag-heading-4 {
+  font-size: var(--denhaag-heading-4-font-size);
+  font-family: var(--denhaag-heading-4-font-family);
+  font-weight: var(--denhaag-heading-4-font-weight);
+  font-style: var(--denhaag-heading-4-font-style);
+  line-height: var(--denhaag-heading-4-line-height);
+  letter-spacing: var(--denhaag-heading-4-letter-spacing);
+}
+
+.denhaag-heading-5 {
+  font-size: var(--denhaag-heading-5-font-size);
+  font-family: var(--denhaag-heading-5-font-family);
+  font-weight: var(--denhaag-heading-5-font-weight);
+  font-style: var(--denhaag-heading-5-font-style);
+  line-height: var(--denhaag-heading-5-line-height);
+  letter-spacing: var(--denhaag-heading-5-letter-spacing);
+}
+
+.denhaag-paragraph {
+  font-size: var(--denhaag-paragraph-font-size);
+  font-family: var(--denhaag-paragraph-font-family);
+  font-weight: var(--denhaag-paragraph-font-weight);
+  font-style: var(--denhaag-paragraph-font-style);
+  line-height: var(--denhaag-paragraph-line-height);
+  letter-spacing: var(--denhaag-paragraph-letter-spacing);
```

Comment:
Do you really need to specify a custom `letter-spacing`? And if Figma says so: does it really look better that way in the browser?

Reply:
I have removed it for now, since I agree this likely won't be used. If the Typography Figma is updated we could add it again.